Examples/Sources/ChatbotExample/Views/ThreadSidebarView.swift

```diff
@@ -0,0 +1,159 @@
+import SwiftCrossUI
+import Foundation
+
+// MARK: - Thread Sidebar View
+
+struct ThreadSidebarView: View {
+    @Binding var threads: [ChatThread]
+    @Binding var selectedThread: ChatThread?
+    
+    let onNewThread: () -> Void
+    let onSelectThread: (ChatThread) -> Void
+    let onDeleteThread: (ChatThread) -> Void
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            // Header
+            HStack {
+                Text("💬 Conversations")
+                    .font(AppFonts.title2)
+                    .fontWeight(.bold)
+                    .foregroundColor(AppColors.text)
+                    .fixedSize(horizontal: true, vertical: false)
+                
+                Spacer()
+            }
+            .padding(.horizontal, AppSpacing.xl)
+            .padding(.vertical, AppSpacing.sm)
+            .background(AppColors.surface)
+        
+            // Thread List
+            ScrollView {
+                VStack(spacing: AppSpacing.sm) {
+                    if threads.isEmpty {
+                        EmptyThreadsView()
+                    } else {
+                        ForEach(threads) { thread in
+                            ThreadRowView(
+                                thread: thread,
+                                isSelected: selectedThread?.id == thread.id,
+                                onSelect: { onSelectThread(thread) },
```

Comment:
I believe that you can just do `onSelect: onSelectThread` to make this a bit cleaner. Same for `onDelete`.